settings.gradle.kts

```
@@ -0,0 +1,8 @@
+pluginManagement {
+    repositories {
+        google()
+        gradlePluginPortal()
+        mavenCentral()
+    }
+}
+rootProject.name = "SqlDelight-js-cipher"
```

SQLDelight writes its own rootProject.names all lowercase.

So for the sake of consistency:

```suggestion
rootProject.name = "sqldelight-js-cipher"
```


Source:
https://github.com/cashapp/sqldelight/search?p=2&q=rootProject&type=code